src/jit/gentree.cpp

```diff
@@ -3045,7 +3045,8 @@ unsigned Compiler::gtSetEvalOrder(GenTree* tree)
     {
         switch (oper)
         {
-            bool iconNeedsReloc;
+            GenTreeIntConCommon* con;
```

Comment:
Is it better to define them in their own scopes?

Reply:
I will make this change in the next chaneet